cdk/lib/unity-license-server-stack.ts

```diff
@@ -56,6 +56,10 @@ export class UnityLicenseServerStack extends cdk.Stack {
         },
       ],
     });
+    eni.applyRemovalPolicy(RemovalPolicy.RETAIN, {
+      applyToUpdateReplacePolicy: true,
+      default: cdk.RemovalPolicy.RETAIN,
+    });
```

Comment:
It should not need the 2nd argument in this case.

```suggestion
    eni.applyRemovalPolicy(RemovalPolicy.RETAIN);
```